### PR TITLE
Rusthello removal

### PR DESCRIPTION
**Pull request by EGhiorzi**
**Problem**: [describe the problem you try to solve with this PR.]

**Solution**: [describe carefully what you change by this PR.]

**Changes introduced by this pull request**:

- [...]
- [...]
- [...]

**Drawbacks**: [if any, describe the drawbacks of this pull request.]

**TODOs**: [what is not done yet.]

**Fixes**: [what issues this fixes.]

**State**: [the state of this PR, e.g. WIP, ready, etc.]

**Blocking/related**: [issues or PRs blocking or being related to this issue.]

**Other**: [optional: for other relevant information that should be known or cannot be described in the other fields.]

------

_The above template is not necessary for smaller PRs._

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -228,7 +224,8 @@ filesystem/bin/%: crates/games/src/%/main.rs crates/games/src/%/*.rs $(BUILD)/cr
 
 games: \
 	filesystem/bin/ice \
-	filesystem/bin/minesweeper
+	filesystem/bin/minesweeper \
+	filesystem/bin/rusthello \
```
Comment (jackpot51) [inline Makefile]: The last line should not have the trailing \

**Comment by EGhiorzi** [comment]
I'm fighting against git and I'm losing. This pull request is probably all wrong

**Comment by jackpot51** [comment]
Everything was ok except for that

**Comment by jackpot51** [comment]
I will fix it

**Comment by EGhiorzi** [comment]
Thanks jackpot! One day I'll manage to use git correctly...

**Pull request merged**